packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/ImsNodeIdField.tsx

```diff
@@ -80,8 +80,8 @@ function ImsNodeId({
         isLoading && locationCode
             ? t('widgets.node_select.nodes_loading')
             : nodes.length
-              ? t('widgets.node_select.select_node')
-              : t('forms.widgets.node_select.no_nodes_placeholder');
+            ? t('widgets.node_select.select_node')
+            : t('forms.widgets.node_select.no_nodes_placeholder');
```

Comment:
This becomes very unreadable, can you try to fix the rule that is responsible for this?